src/twix.coffee

```diff
@@ -514,4 +514,8 @@ module.exports = makeTwix(require "moment") if hasModule
 if  typeof(define) == "function"
   define "twix", ["moment"], (moment) -> makeTwix(moment)
 
-@Twix = makeTwix(@moment) if @moment?
+if @moment
+  @Twix = makeTwix(@moment)
+else if moment?
+  # Also checks globals (Meteor)
+  @Twix = makeTwix(moment)
```

Comment:
Sorry to keep going back and forth on this -- I haven't used Meteor and I'm not used to not being able to set globals with `@`. But one last idea, because I really don't like this create-and-then-delete thing. Can we just do `Twix = makeTwix(moment)` in the `moment?` case (I assume that's sufficient to create a global in Meteor)? If so, then the locale part could just use the global Twix.